src/cnn_feature_extractor/README.txt

```diff
@@ -0,0 +1,18 @@
+
+Using Caffe (http://caffe.berkeleyvision.org/), cnn_feature_extractor extracts a feature vector from each input image. It is assumed Caffe has been built and the header files and library (libcaffe.so) are available.
+
+To run the program, first download two files
+bvlc_reference_caffenet.caffemodel
+imagenet_mean.binaryproto
+from Caffe and copy them to
```

Comment:
When you say `from Caffe`, I'm not seeing an obvious place where these are coming from on their site.